rollup.config.js

```diff
@@ -3,17 +3,23 @@ import resolve from '@rollup/plugin-node-resolve';
 import commonjs from '@rollup/plugin-commonjs';
 import typescript from 'rollup-plugin-typescript2';
 import postcss from 'rollup-plugin-postcss';
+import json from '@rollup/plugin-json'; //required for react-syntax-highlighter package in CodeFence.tsx
 
 const packageJson = require('./package.json');
 
 export default {
 	input: packageJson.exports.require,
 	output: [
 		{
-			file: packageJson.exports.default,
+			dir: packageJson.exports.default,
 			format: 'esm',
 			sourcemap: true,
 		},
 	],
-	plugins: [peerDepsExternal(), resolve(), commonjs(), typescript({ useTsconfigDeclarationDir: true }), postcss()],
+	plugins: [peerDepsExternal(),
+		resolve(),
+		commonjs(),
+		typescript({ useTsconfigDeclarationDir: true }),
+		json(),
```

Comment:
Don't forget to comment this one about what purpose it serves. It's not obvious because the project neither directly contains nor directly outputs any JSON files.

Reply:
I commented on the line where the file was imported